src/Sharpliner/AzureDevOps/ConditionedExpressions/Interfaces/ICompileTimeExpression.cs

```diff
@@ -0,0 +1,6 @@
+namespace Sharpliner.AzureDevOps.ConditionedExpressions.Interfaces;
+
+public interface ICompileTimeExpression
```

Comment:
Could you add a short XML doc to each of these and what they represent? It might be helpful for future

Reply:
Done